dist.ini

```
@@ -0,0 +1,108 @@
+; REF: Dist::Zilla https://metacpan.org/pod/Dist::Zilla
+name = Crypt-OpenSSL-PKCS12
+author = Dan Sully
+main_module = PKCS12.pm
+license = Perl_5
+copyright_holder = Dan Sully
+copyright_year = 2004-2021
```

Please bump the year

```suggestion
copyright_year = 2004-2024
```